packages/nextjs/src/store/postsStore.ts

```
@@ -0,0 +1,195 @@
+  setSearchQuery: (query: string) => void;
+  searchPosts: (query: string) => void;
+  clearSearch: () => void;
+  addPost: (post: PostProps) => void;
+}
+
+// Function to normalize text (remove accents and convert to lowercase)
+const normalizeText = (text: string): string => {
+  return text
+    .toLowerCase()
+    .normalize('NFD')
+    .replace(/[\u0300-\u036f]/g, '') // Remove accents and diacritics
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)